File: CSharp/leetcodesln/ConvertIntegerToTheSumOfTwoNo-ZeroIntegers.cs
using System;

namespace leetcodesln
{
    public class ConvertIntegerToTheSumOfTwoNo_ZeroIntegers
    {
        public int[] GetNoZeroIntegers(int n)
        {
            var ans = new int[2];

            for (int i = 0; i < n; i++)
            {
                if (!i.ToString().Contains("0"))
                {
                    var comp = n - i;
                    if (!comp.ToString().Contains("0"))
                    {
                        ans[0] = i;
                        ans[1] = comp;
                        return ans;
                    }
                }
            }
            throw new Exception();
        }
    }
}

File: Go/questions/ConvertIntegerToTheSumOfTwoNo-ZeroIntegers.go
package questions

import (
	"strconv"
	"strings"
)

func GetNoZeroIntegers(n int) []int {
	ans := make([]int, 0)

	for i := 1; i < n; i++ {
		a := strconv.Itoa(i)
		if !strings.Contains(a, "0") {
			comp := n - i
			b := strconv.Itoa(comp)
			if !strings.Contains(b, "0") {
				ans = append(ans, i)
				ans = append(ans, comp)
				return ans
			}
		}
	}
	return ans
}
